packages/client-core/src/components/World/LoadLocationScene.tsx

```diff
@@ -88,10 +88,15 @@ export const useLoadLocation = (props: { locationName: string }) => {
 export const useLoadScene = (props: { projectName: string; sceneName: string }) => {
   const sceneKey = `projects/${props.projectName}/${props.sceneName}`
   const assetID = useFind(staticResourcePath, { query: { key: sceneKey, type: 'scene' } })
+  const sceneID = useHookstate<string | undefined>(undefined)
```

Comment:
what's the reason for adding this?

Reply:
The CMS changes broke the scene loading in the dev test suite, it hook into the LocationState and get the id that way, but it feels very unintuitive to have loading a scene be a side effect

Reply:
hmm we should talk about this. i think i fixed it in a recent PR